src/config/loader.py

```diff
@@ -1,108 +1,73 @@
-                return_value = self._parsers["custom"][section][key]
-            else:
-                return_value = self._parsers["params"][section][key]
-            return return_value
-        except KeyError:
-            Logger.error(f"Key '{key}' not found in section '{section}'")
-            os._exit(1)
-
-    def _load_params(self):
-        self._parsers["params"] = configparser.ConfigParser()
-        self._parsers["custom"] = configparser.ConfigParser()
-        self._parsers["params"].read(self._config_path / PARAMS_INI, encoding="utf-8")
-        if (p := (self.user_dir / PARAMS_INI)).exists():
-            if p.stat().st_size:
-                self._parsers["custom"].read(p, encoding="utf-8")
-        else:
+    def load(self, clear: bool = False):
+        if not (self.user_dir / PARAMS_INI).exists() or clear:
             with open(self.user_dir / PARAMS_INI, "w", encoding="utf-8"):
                 pass
 
-        self._advanced_options = AdvancedOptionsModel(
-            exit_key=self._select_val("advanced_options", "exit_key"),
-            log_lvl=self._select_val("advanced_options", "log_lvl"),
-            move_to_chest=self._select_val("advanced_options", "move_to_chest"),
-            move_to_inv=self._select_val("advanced_options", "move_to_inv"),
-            process_name=self._select_val("advanced_options", "process_name"),
-            run_filter=self._select_val("advanced_options", "run_filter"),
-            run_filter_force_refresh=self._select_val("advanced_options", "run_filter_force_refresh"),
-            run_scripts=self._select_val("advanced_options", "run_scripts"),
-            scripts=self._select_val("advanced_options", "scripts").split(","),
-        )
-        self._char = CharModel(inventory=self._select_val("char", "inventory"))
-        self._general = GeneralModel(
-            browser=self._select_val("general", "browser"),
-            check_chest_tabs=self._select_val("general", "check_chest_tabs").split(","),
-            full_dump=self._select_val("general", "full_dump"),
-            handle_rares=self._select_val("general", "handle_rares"),
-            hidden_transparency=self._select_val("general", "hidden_transparency"),
-            keep_aspects=self._select_val("general", "keep_aspects"),
-            move_to_inv_item_type=self._select_val("general", "move_to_inv_item_type"),
-            move_to_stash_item_type=self._select_val("general", "move_to_stash_item_type"),
-            profiles=self._select_val("general", "profiles").split(","),
-            run_vision_mode_on_startup=self._select_val("general", "run_vision_mode_on_startup"),
-        )
+        self._parser = configparser.ConfigParser()
+        self._parser.read(self.user_dir / PARAMS_INI, encoding="utf-8")
+
+        if "advanced_options" in self._parser:
+            self._advanced_options = AdvancedOptionsModel(**self._parser["advanced_options"])
```

Comment:
I really like this because now new properties don't need to be added in multiple places